frontend/src/pages/Seller/Auth/CreateAccount.tsx

```diff
@@ -1,9 +1,86 @@
 import * as React from "react"
+import styles from "./SellerRegister.module.css";
+import { Input } from '@chakra-ui/react';
+import { Button, ButtonGroup } from '@chakra-ui/react'
+import {useState} from "react";
+import { Link } from '@chakra-ui/react'
+import orangeGradient from'../../../assets/orangeGradient.png';
```

Comment:
This throws an error when I boot up the frontend. I don't see an `orangeGradient.png` image in the PR

Reply:
See instructions here https://github.com/miguelaenlle/merch-order-request-form/pull/18/files#r1513613047 to commit `orangeGradient`